### ISSUE TITLE
Error when trying to update load-balancer-outbound-ips with aks-preview extension

### ISSUE THREAD

**Issue by fschim** [Azure/azure-cli-extensions#4205]
### Describe the bug
---
Fail to update load-balancer-outbound-ips with aks-preview extension installed
```
# az version
{
  "azure-cli": "2.31.0",
  "azure-cli-core": "2.31.0",
  "azure-cli-telemetry": "1.0.6",
  "extensions": {
    "aks-preview": "0.5.47"
  }
}
# az aks update --resource-group {} --subscription {} --name {} --load-balancer-outbound-ips {} --debug
...
cli.azure.cli.core.sdk.policies: Response content:
cli.azure.cli.core.sdk.policies: {
  "code": "InvalidLoadBalancerProfile",
  "message": "Load balancer profile must specify exactly one of ManagedOutboundIPs, OutboundIPPrefixes and OutboundIPs. Please refer to http://aka.ms/slb.",
  "target": "networkProfile.loadBalancerProfile"
 }
cli.azure.cli.core.util: azure.cli.core.util.handle_exception is called with an exception:
cli.azure.cli.core.util: Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/knack/cli.py", line 231, in invoke
    cmd_result = self.invocation.execute(args)
  File "/usr/local/lib/python3.9/site-packages/azure/cli/core/commands/__init__.py", line 658, in execute
    raise ex
  File "/usr/local/lib/python3.9/site-packages/azure/cli/core/commands/__init__.py", line 721, in _run_jobs_serially
    results.append(self._run_job(expanded_arg, cmd_copy))
  File "/usr/local/lib/python3.9/site-packages/azure/cli/core/commands/__init__.py", line 692, in _run_job
    result = cmd_copy(params)
  File "/usr/local/lib/python3.9/site-packages/azure/cli/core/commands/__init__.py", line 328, in __call__
    return self.handler(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/azure/cli/core/commands/command_operation.py", line 121, in handler
    return op(**command_args)
  File "/root/.azure/cliextensions/aks-preview/azext_aks_preview/custom.py", line 1233, in aks_update
    return _put_managed_cluster_ensuring_permission(cmd,
  File "/root/.azure/cliextensions/aks-preview/azext_aks_preview/custom.py", line 3001, in _put_managed_cluster_ensuring_permission
    cluster = sdk_no_wait(no_wait, client.begin_create_or_update,
  File "/usr/local/lib/python3.9/site-packages/azure/cli/core/util.py", line 646, in sdk_no_wait
    return func(*args, **kwargs)
  File "/root/.azure/cliextensions/aks-preview/azext_aks_preview/vendored_sdks/azure_mgmt_preview_aks/v2021_10_01/operations/_managed_clusters_operations.py", line 736, in begin_create_or_update
    raw_result = self._create_or_update_initial(
  File "/root/.azure/cliextensions/aks-preview/azext_aks_preview/vendored_sdks/azure_mgmt_preview_aks/v2021_10_01/operations/_managed_clusters_operations.py", line 686, in _create_or_update_initial
    raise HttpResponseError(response=response, error_format=ARMErrorFormat)
azure.core.exceptions.HttpResponseError: (InvalidLoadBalancerProfile) Load balancer profile must specify exactly one of ManagedOutboundIPs, OutboundIPPrefixes and OutboundIPs. Please refer to http://aka.ms/slb.
Code: InvalidLoadBalancerProfile
Message: Load balancer profile must specify exactly one of ManagedOutboundIPs, OutboundIPPrefixes and OutboundIPs. Please refer to http://aka.ms/slb.
Target: networkProfile.loadBalancerProfile
...
```

However once `aks-preview` extension is removed, the same command could be executed without error.

### Expected Behavior
---
Command should not fail even with `aks-preview` extension installed.

### Environment Summary
---
Azure CLI docker
```
docker run -it mcr.microsoft.com/azure-cli:2.31.0
```

**Comment by jsntcy** [comment]
Route to service team

**Comment by FumingZhang** [comment]
Hi @fschim, sorry for the inconvenience, the issue has been fixed in v0.5.48 ([PR #4232](https://github.com/Azure/azure-cli-extensions/pull/4232)), please update to the latest version via `az extension update -n aks-preview`.

**Comment by fschim** [comment]
@FumingZhang Thanks for the quick response.
Tried with aks-preview 0.5.48 and the command could go through now without error.
